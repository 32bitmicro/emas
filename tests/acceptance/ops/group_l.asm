LD r0
LF r0
LA r0
LL r0
TD r0
TF r0
TA r0
TL r0
