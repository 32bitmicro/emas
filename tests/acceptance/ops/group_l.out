@ 0x0000 : 0xf400  /  111 101 0 000 000 000  /  62464
@ 0x0001 : 0xf440  /  111 101 0 001 000 000  /  62528
@ 0x0002 : 0xf480  /  111 101 0 010 000 000  /  62592
@ 0x0003 : 0xf4c0  /  111 101 0 011 000 000  /  62656
@ 0x0004 : 0xf500  /  111 101 0 100 000 000  /  62720
@ 0x0005 : 0xf540  /  111 101 0 101 000 000  /  62784
@ 0x0006 : 0xf580  /  111 101 0 110 000 000  /  62848
@ 0x0007 : 0xf5c0  /  111 101 0 111 000 000  /  62912
